src/app/collective-rewards/utils/isBuilderOperational.ts

```diff
@@ -17,6 +17,16 @@ export const isBuilderActive = (stateFlags?: BuilderStateFlags) => {
   )
 }
 
+export const isBuilderRewarded = (stateFlags?: BuilderStateFlags) => {
```

Comment:
I don't get why this reward is linked to the state. Should we rename it to `isBuilderRewardable`?